server/app/middlewares/websocket_backend.rb

```diff
@@ -237,23 +237,22 @@ def subscribe_to_rpc_channel
 
   # @param [Hash] msg
   def on_rpc_message(msg)
-    client = client_for_id(msg['id'])
-    if client
-      self.send_message(client[:ws], msg['message'])
-    end
+    EM.next_tick{
+      client = client_for_id(msg['id'])
+      if client
+        self.send_message(client[:ws], msg['message'])
+      end
+    }
   rescue => exc
```

Comment:
Does `EM.next_tick` propagate exceptions? I doubt it, and I suspect this doesn't rescue anything anymore?